README.md

```diff
@@ -56,33 +57,63 @@ export class UserController {
 
 ## API
 
-### create(initialContext?: object, domain? :string)
+### run(fn: Function, context: unknown)
 
-Creates for the current async resource an execution context entry identified with his asyncId.
-Any future processes that will be added to the async execution chain will be exposed to this context.
+Runs given callback that will be exposed to the given context.
+The context will be exposed within all callbacks and promises chain that will be trigger from given fn.
 
-> When passing custom domain to this method, the trigger point and all of it's sub processes will be exposed to a standalone context and won't effect / be effected by root context. 
+### get()
 
-### update(update: object)
+Gets the current asynchronous execution context.
 
-Updates the current execution context with a given update obect.
+> This API may throw CONTEXT_DOES_NOT_EXISTS error if accessed without initializing the context properly.
```

Comment:
change const to `CONTEXT_DOES_NOT_EXIST`

Reply:
you mean regarding the fact that the error message is not `CONTEXT_DOES_NOT_EXIST`?